Automattic/wp-calypso
# add hosting trial product to plans list

Closes https://github.com/Automattic/dotcom-forge/issues/4043

## Proposed Changes

* Add hosting trial plan when `plans/hosting-trial `is enabled
* Add hosting trial plan to available list for:
1.  `PLAN_BUSINESS`
2. `PLAN_BUSINESS_2_YEARS`
3. `PLAN_BUSINESS_3_YEARS`
4. `PLAN_ECOMMERCE`
5. `PLAN_ECOMMERCE_MONTHLY`
6. `PLAN_ECOMMERCE_2_YEARS`
7. `PLAN_ECOMMERCE_3_YEARS`

## Testing Instructions



*  Force plan at  `checkout/siteSlug/wp_bundle_hosting_trial_monthly`. You should see the plan name in the cart.
![image](https://github.com/Automattic/wp-calypso/assets/47489215/29789dbe-d205-43e6-9a74-82d21ee4781a)


## Pre-merge Checklist
